src/OrbitGl/ThreadTrack.cpp

```diff
@@ -421,9 +420,8 @@ void ThreadTrack::OnTimer(const TimerInfo& timer_info) {
   // Thread tracks use a ScopeTree so we don't need to create one TimerChain per depth.
   // Allocate a single TimerChain into which all timers will be appended.
 
-  // Pass ownership to timer_chain. TODO(http://b/194268477): Pass timer_info as a value instead of
-  // reference to be able to move it.
```

Comment:
It seems to me this TODO has not been applied yet. `ThreadTrack::OnTimer` still gets TimerInfo by const-ref.

AFAIK the scope of the bug was to change the function signature of `TimerTrack::OnTimer` to accept a value instead of a const-ref.

Reply:
restored.